crypto.go

```diff
@@ -391,12 +566,20 @@ func (s sikeScheme) GenerateKeyPair(rand io.Reader) (KEMPrivateKey, KEMPublicKey
 }
 
 func (s sikeScheme) Marshal(pk KEMPublicKey) []byte {
+	if pk == nil {
+		return nil
+	}
 	raw := pk.(*sikePublicKey)
 	out := make([]byte, raw.pub.Size())
 	raw.pub.Export(out)
 	return out
 }
 
+func (s sikeScheme) MarshalPrivate(sk KEMPrivateKey) []byte {
+	// There's no standard way to encode a SIKE private key, so return nil (for now).
```

Comment:
Don't really care about "standard" here, since we could just specify it in the document.  More salient that we we're not going to generate test vectors right now.

Reply:
I wonder if these unimplemented methods should panic instead of returning.  Fail fast.

Reply:
Yep, let's panic for now.